app/controllers/ticket_controller.rb

```diff
@@ -54,13 +54,13 @@ def make_reservation
     end
 
     def confirm_reservation
-      Ticket.where(:paid => false).update_all(paid: true)
+      Ticket.where(paid: false).update_all(paid: true)
       render :json => "You've paid for ticket that You had booked."
     end
 
     def ticket_available?
       @cinema_hall = CinemaHall.find(params[:cinema_hall_id])
-      Ticket.where(cinema_hall_id: params[:cinema_hall_id], paid: true).count(:all) < @cinema_hall.read_attribute_before_type_cast(:volume) 
+      Ticket.where(params[:movies_id]).count(:all) < @cinema_hall.read_attribute_before_type_cast(:volume) 
```

Comment:
What should be the logic here?

Reply:
You can buy ticket If counter of all tickets in database with movies_id reaches below volume of the cinema_hall. 
If counter == volume, you cant buy

Reply:
1. You don't need to pass `:all` to `count` method
2. Why do you need `read_attribute_before_type_cast` here? AFAIK it would return string but what you need here is a number